### PR TITLE
Various: Process duplicate region/dependency records

### LINKED ISSUE #988: Fix duplicate dependency/region records
Often times, `dependency` records have a single descendant `region` record. Examples:

**Guam**
Dependency: https://whosonfirst.mapzen.com/spelunker/id/85632163/
Region: https://whosonfirst.mapzen.com/spelunker/id/85681227/

**Puerto Rico**
Dependency: https://whosonfirst.mapzen.com/spelunker/id/85633729/
Region: https://whosonfirst.mapzen.com/spelunker/id/85687331/

These `region` records are duplicates - we should maintain a single record for each of these places, but tag the `dependency` record with a placetype_alt...

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #988.

- Adds placetype_alt of "region" to dependencies
- Supersedes any region record into its duplicate dependency record

To do:

- [x] Save region geoms as alt geoms for the dependency record
- [x] Save duplicate concordances in dependency record
- [x] Bring over any missing names from the region record to the dependency record.

### CODE REVIEW

**Comment by stepps00** [comment]
Notes for the first "to do" ([commit]()):

dependency_id,region_id
85632629,85681419 (new alt geom of region)
85632517,85681311 (swapped region geom in for dependency, saved dependency as alt)
85633167,85684619 (swapped region geom in for dependency, saved dependency as alt)
85632689,85681165 (new alt geom of region)
85632301,85681367 (new alt geom of region)
85632587,85681325 (region geom already stored as an alt geom)
85632617,85681173 (new alt geom of region)
85633729,85687331 (region geom already stored as an alt geom)
85632387,85671119 (new alt geom of region)
85632199,85669283 (new alt geom of region)
85632459,85681249 (region geom already stored as an alt geom)
85632493,85681315 (region geom already stored as an alt geom)
85632239,85681231 (region geom already stored as an alt geom)
85632613,85681151 (keeping as is, the region geometry is not worth saving)
85632649,85681409 (region geom already stored as an alt geom)
85632743,85680913 (region geom already stored as an alt geom)
85632163,85681227 (region geom already stored as an alt geom)
85632211,85681209 (region geom already stored as an alt geom)
85632611,85681183 (region geom already stored as an alt geom)

**Comment by stepps00** [comment]
Notes for the second commit ([commit](4980c90)):

Saved any concordance values from the duplicate region record into the dependency record.

File: data/856/321/99/85632199.geojson [hunk 1]
```diff
@@ -465,6 +466,9 @@
         "wof:parent_id"
     ],
     "wof:country":"BL",
+    "wof:cponcordance_alt":{
```
Comment (stepps00) [inline data/856/321/99/85632199.geojson]: CONCORDANCE

File: data/856/321/99/85632199.geojson [hunk 2]
```diff
@@ -437,7 +437,8 @@
     "qs:source":"US State Department, with Natural Earth mods",
     "src:geom":"quattroshapes",
     "src:geom_alt":[
-        "naturalearth"
+        "naturalearth",
+        "quattroshapes"
```
Comment (nvkelso) [inline data/856/321/99/85632199.geojson]: Isn't this odd that `quattroshapes` is listed as `src:geom` and in `src:geom_alt`? I guess the alt has a less precise geom, so we could do the work to figure out what "zoom" modifier to add to the file name. Eh, skip.

**Comment by stepps00** [comment]
This is cleared to import, so the most recent commit merges in `master` to remove conflicts and verifies new property types. Should be ready to merge now.

Pinging @nvkelso in case you'd like to review once more.

**Comment by nvkelso** [comment]
LGTM

**Pull request merged**